test_scripts/WebEngine/commonWebEngine.lua

```diff
@@ -237,6 +237,65 @@ function common.activateWidgetFromNoneToFULL(pId, pAppId)
   checkAbsenceOfOnHMIStatusForOtherApps(pAppId)
 end
 
+function common.getShowParams(pAppId)
+  return {
+    requestShowParams = {
+      mainField1 = "Text_1",
+      graphic = {
+        imageType = "DYNAMIC",
+        value = "icon.png"
+      }
+    },
+    requestShowUiParams = {
+      showStrings = {
+        {
+          fieldName = "mainField1",
+          fieldText = "Text_1"
+        }
+      },
+      graphic = {
+        imageType = "DYNAMIC",
+        value = actions.getPathToFileInStorage("icon.png", pAppId)
+      }
+    }
+  }
+end
+
+function common.sendShowToWindow(pWindowId, pAppId)
+  if not pAppId then pAppId = 1 end
+  local params = common.getShowParams(pAppId)
+  if pWindowId then
+    params.requestShowParams.windowID = pWindowId
+    params.requestShowUiParams.windowID = pWindowId
+  end
+  params.requestShowUiParams.appID = common.getHMIAppId(pAppId)
+  local cid = common.getMobileSession(pAppId):SendRPC("Show", params.requestShowParams)
+  common.getHMIConnection():ExpectRequest("UI.Show", params.requestShowUiParams)
+  :ValidIf(function(_,data)
+      if pWindowId == nil and data.windowID ~= nil then
+        return false, "SDL sends not exist window ID to HMI"
+      else
+        return true
+      end
+    end)
+  :Do(function(_, data)
+      common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {})
+    end)
+  common.getMobileSession(pAppId):ExpectResponse(cid, { success = true, resultCode = "SUCCESS" })
+  :Do(function()
+      if params.requestShowParams.templateConfiguration ~= nil then
+        local paramsToSDL = common.getOnSystemCapabilityParams()
+        paramsToSDL.appID = common.getHMIAppId(pAppId)
+        common.getHMIConnection():SendNotification("BasicCommunication.OnSystemCapabilityUpdated", paramsToSDL)
+        common.getMobileSession(pAppId):ExpectNotification("OnSystemCapabilityUpdated",
+          common.getOnSystemCapabilityParams())
```

Comment:
@VjKlepikov I think that _getOnSystemCapabilityParams_ function should be updated... Here https://github.com/LuxoftSDL/sdl_atf_test_scripts/blob/1016b4a3ffbf90cf4aefdfa1edb3a51ce6ed5679/test_scripts/WebEngine/commonWebEngine.lua#L516 we should specify WEB_VIEW template as available. Could you please check it

Reply:
@LitvinenkoIra getOnSystemCapabilityParams contains update for  widget (windowID = 1). According to propousal
the WEB_VIEW template is available only to the main window. It should not be available for widgets. So we shouldn't specify WEB_VIEW template as available for the widget (windowID = 1)